src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -50,19 +50,19 @@ public class VoucherManagementStepDef extends BaseStepDef {
 
     @Given("I can create an VoucherRequestDTO for voucher creation")
     public void iCreateAnIdentityMapperDTOForRegisterBeneficiary() {
-        requestId = generateUniqueNumber(16);
+        requestId = generateUniqueNumber(12);
         StringBuilder sb = new StringBuilder();
         sb.append("{\n");
         sb.append("    \"requestID\": \"").append(requestId).append("\",\n");
         sb.append("    \"batchID\": \"").append(generateUniqueNumber(10)).append("\",\n");
         sb.append("    \"voucherInstructions\": [\n");
         sb.append("        {\n");
-        sb.append("            \"instructionID\": \"").append(generateUniqueNumber(18)).append("\",\n");
+        sb.append("            \"instructionID\": \"").append(generateUniqueNumber(16)).append("\",\n");
         sb.append("            \"groupCode\": \"021\",\n");
         sb.append("            \"currency\": \"SGD\",\n");
-        sb.append("            \"amount\": 3000,\n");
-        sb.append("            \"payeeFunctionalID\": \"08187379158030059976\",\n");
-        sb.append("            \"narration\": \"Social Support Payment for the Month of January 2023\"\n");
+        sb.append("            \"amount\": 9000,\n");
+        sb.append("            \"payeeFunctionalID\": \"63310590322288932682\",\n");
```

Comment:
Is this a negative scenario? Why this Id is hard -coded?

Do not create the request body using String, create a POJO and then convert that to JSON. We did this approach earlier and ended up with loads of technical debt.